packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -127,6 +128,21 @@ Future<void> buildWindows(
     'Built ${globals.fs.path.relative(buildOutput.path)}',
     color: TerminalColor.green,
   );
+  final File shorebirdYamlFile = buildDirectory
+    .childDirectory('runner')
+    .childDirectory(sentenceCase(buildModeName))
+    .childDirectory('data')
+    .childDirectory('flutter_assets')
+    .childFile('shorebird.yaml');
```

Comment:
I'm a little surprised there isn't a flutterAssets() path helper somewhere.

Reply:
Me too. Searching for "assets" in either cmake or windows files yielded nothing though 🤔